Microsoft.Toolkit.Uwp.SampleApp/SamplePages/MarkdownTextBlock/InitialContent.md

```diff
@@ -1,5 +1,7 @@
 ﻿This control was originally written by [Quinn Damerell](https://github.com/QuinnDamerell) and [Paul Bartrum](https://github.com/paulbartrum) for [Baconit](https://github.com/QuinnDamerell/Baconit), a popular open source reddit UWP. The control *almost* supports the full markdown syntax, with a focus on super-efficient parsing and rendering. The control is efficient enough to be used in virtualizing lists.
 
+<!-- Comments are now Implemented -->
```

Comment:
Add an example of what a comment would look like above this, such as:
````
Comments can be added in Markdown, and they won't be rendered to the screen.

To create a comment, enclose in XML style comment tags:
```
<!-- Comments are now Implemented -->
```
There is a Comment below this line.
<!-- Comments are now Implemented -->
````

Reply:
@WilliamABradley I will update the PR over the weekend.